pkg/services/dashboards/database/database.go

```diff
@@ -385,15 +385,15 @@ func getExistingDashboardByIDOrUIDForUpdate(sess *db.Session, dash *dashboards.D
 	return isParentFolderChanged, nil
 }
 
+// Returns boolean on whether the parent folder changed. Error is if the dashboard already exists.
```

Comment:
thoughts on this documentation comment?

Reply:
Linting may fail on this in CI.  If not don't want to block shipping this but usually it's best practice to document functions like this:
`// getExistingDashboardByTitleAndFolder returns a boolean (on whether the parent folder changed) and an error for if the dashboard already exists.`

Reply:
ahhh, noted - thank you. will implement the comment like that